bitnami/kube-prometheus/values.yaml

```diff
@@ -297,13 +335,26 @@ prometheus:
     # annotations:
     #   eks.amazonaws.com/role-arn: arn:aws:iam::ACCOUNT:role/prometheus
 
-  ## SecurityContext configuration
+  ## Prometheus pods' Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
   ##
-  securityContext:
+  podSecurityContext:
```

Comment:
I don't know if it is smart to rename here.

Reply:
this implies to all the `values.yaml` securityContext -> podSecurityContext renames.

Reply:
I think that you can keep using podSecurityContext in this case. I mean, this change is a major change. You are deprecating some parameters to use new ones and I think that we are not going to have retro-compatibility.

Reply:
Inside the prometheus and alertmanager manifests it is still named `securityContext` not `podSecurityContext`. This is why I asked ^^

Reply:
wow, that it is true. I think that the best option is to use podSecurityContext but if we choose this way, we need to change the other ones too. Keep going using podSecurityContext and we can create another batch of patches modifying the other ones.

Reply:
that is the point I wanted to make. It will not change in the `prometheus.yaml` manifest as of now.

Reply:
perfect, thanks @Cellebyte